flex-templates/python/regional_dlp_transform/bigquery_dlp_bigquery.py

```diff
@@ -103,37 +119,68 @@ def run(argv=None, save_main_session=True):
     with beam.Pipeline(options=options) as p:
 
         # Read from BigQuery into a PCollection.
-        messages = (
-            p
-            | 'Read from BigQuery Table' >>
-            beam.io.ReadFromBigQuery(
-                table=known_args.input_table
+        if known_args.query is not None:
+            messages = (
+                p
+                | 'Read from BigQuery Table' >>
```

Comment:
I'd suggest changing title to something different from the input table case.  maybe indicate it's using query to read from BQ table.